src/components/HyperlinkingWrapper.js

```diff
@@ -160,6 +182,13 @@ export default class HyperlinkingWrapper extends Component {
         };
     }
 
+    formatLink(url) {
+        const { blotToEdit } = this.state;
+        const objectToFormat = blotToEdit || this.quill;
+
+        objectToFormat.format('link', url || undefined);
```

Comment:
on line #153 you call it with `undefined` which results in `undefined || undefined`

do we need that fallback? do we need to call this func with `undefined`?